package.json

```diff
@@ -36,7 +40,8 @@
     "react-native-screens": "~2.15.2",
     "react-native-svg": "^12.1.0",
     "react-native-web": "~0.13.12",
-    "url-otpauth": "^2.0.0"
+    "url-otpauth": "^2.0.0",
+    "uuid-random": "1.3.2"
```

Comment:
prefer the more popular https://www.npmjs.com/package/uuid

Reply:
Done!